### DISCUSSION TITLE
create course in multiplayer

### DISCUSSION THREAD

**Discussion by lordroshi** [Courseplay/Courseplay_FS22#2288] [Q&A]
hi, we have a multiplayer game with couseplay. one of the players has a problem, when he tries to create a field he gets kicked out of the game every time. only one player has this problem, the others don't. what can be the reason for this?
![20230205121544_1](https://user-images.githubusercontent.com/124498954/216815662-ff227635-d8ea-46c7-819e-c5934902bb23.jpg)
![20230205121558_1](https://user-images.githubusercontent.com/124498954/216815668-d20b68c2-4f05-40b6-be06-c02125c304af.jpg)
![20230205121605_1](https://user-images.githubusercontent.com/124498954/216815674-e741b8c5-2e9d-44e2-96cd-094beb0e7856.jpg)
![20230205121627_1](https://user-images.githubusercontent.com/124498954/216815681-26d9df59-851b-4e6a-9892-9aca70d280f4.jpg)

**Comment by Tensuko** [comment]
Good question.
Logfiles from server and client when this happens.
Not sure if a bad internet connection can couse this.

**Comment by lordroshi** [comment]
[https://drive.google.com/drive/folders/1FWPowi7aGnoXHgB_9t2UmxY2gDxqQCq9?usp=sharing](url)

the internet connection seems to be fine. We played 100h+ on the server with no issues.

**Comment by Tensuko** [comment]
The amount of playtime says nothing about the internet connection during maybe high load.

**Comment by lordroshi** [comment]
The Problem ist still there. Testet it at different times. I Put the logs in Google Drive. The Link above. 

Also reinstalled the entire Game + all Data in document folder from the Client.

Wer tried the Savegame in sp, No problems with CP. Then tried a mp-game on a normal PC, No Problem with CP. Saved the game in the Client and uploaded back on the Server. Same problem with field creator as in the first post.

The Server ist a Rental Server. Hope this helps.

**Comment by daniloqz** [comment]
The same problem happens to me, I rented a server on Gameservers.com, when I upload Courseplay, "Issues - file count per store too high (max 128)" appears, that is, dedicated servers accept up to 128 files and Courseplay go beyond that. When I test in game, I can open the menu, but when I go to save or generate a route, I am disconnected from the game like you said and showed, I even did the same tests and I see that it only occurs on the dedicated server. It seems to be a problem that the mod developer was unable to resolve, as I play on fs19 and have yet to see anyone resolve the issue. I am playing in FS19.

**Reply by Tensuko** [reply]
That file count error have no effect on anything and no need to be resolved. Else on one would be able to use CP on MP Servers.

Disconnection must have to do with anything else but since only a few player have this with no real error when disconnect or any sign of a problem, there is no chanze to find out why this happens.
